bundles/net.tourbook.export/format-templates/gpx-1.0.vm

```diff
@@ -47,11 +47,11 @@
          <mt:conconiDeflection>$tourData.ConconiDeflection</mt:conconiDeflection>
          <mt:dpTolerance>$tourData.DpTolerance</mt:dpTolerance>
 ## weather
-         <mt:temperature>$tourData.AvgTemperature</mt:temperature>
+         <mt:temperature>$tourData.weather_Temperature_Average_Device</mt:temperature>
```

Comment:
@wolfgang-ch 
Thanks to the unit tests 😄  that failed, I was able to update this file, otherwise I would have broken this export GPX feature.